tensorly/backend/mxnet_backend.py

```diff
@@ -51,110 +49,27 @@ def ndim(tensor):
         return tensor.ndim
 
     @staticmethod
-    def reshape(tensor, shape):
-        if not shape:
-            shape = [1]
-        return nd.reshape(tensor, shape)
-
-    def solve(self, matrix1, matrix2):
-        ctx = self.context(matrix1)
-        matrix1 = self.to_numpy(matrix1)
-        matrix2 = self.to_numpy(matrix2)
-        res = numpy.linalg.solve(matrix1, matrix2)
-        return self.tensor(res, **ctx)
-
-    @staticmethod
-    def min(tensor, *args, **kwargs):
-        if isinstance(tensor, nd.NDArray):
-            return nd.min(tensor, *args, **kwargs).asscalar()
-        else:
-            return numpy.min(tensor, *args, **kwargs)
-
-    @staticmethod
-    def max(tensor, *args, **kwargs):
-        if isinstance(tensor, nd.NDArray):
-            return nd.max(tensor, *args, **kwargs).asscalar()
-        else:
-            return numpy.max(tensor, *args, **kwargs)
-
-    @staticmethod
-    def argmax(data=None, axis=None):
-        res = nd.argmax(data, axis)
-        if res.shape == (1,):
-            return res.astype('int32').asscalar()
-        else:
-            return res
-
-    @staticmethod
-    def argmin(data=None, axis=None):
-        res = nd.argmin(data, axis)
-        if res.shape == (1,):
-            return res.astype('int32').asscalar()
```

Comment:
I had added these `asscalar` and typecasting where appropriate to compensate for the issue with 0-order tensors. Do you think it's better to remove them all?